<29>Jul  3 21:42:57 openvpn[66026]: 10.170.120.149:37849 peer info: IV_VER=3.git:released:662eae9a:Release
<29>Jul  3 21:42:57 openvpn[66026]: 10.170.120.149:37849 peer info: IV_PLAT=android
<29>Jul  3 21:42:57 openvpn[66026]: 10.170.120.149:37849 peer info: IV_NCP=2
<29>Jul  3 21:42:57 openvpn[66026]: 10.170.120.149:37849 peer info: IV_TCPNL=1
<29>Jul  3 21:42:57 openvpn[66026]: 10.170.120.149:37849 peer info: IV_GUI_VER=net.openvpn.connect.android_3.2.4-5891
<29>Jul  3 21:42:57 openvpn[66026]: 10.170.120.149:37849 peer info: IV_PROTO=2
<29>Jul  3 21:42:57 openvpn[66026]: 10.170.120.149:37849 peer info: IV_SSO=openurl
<29>Jul  3 21:42:57 openvpn[66026]: 10.170.120.149:37849 [bob] Peer Connection Initiated with [AF_INET]10.170.120.149:37849
<37>Jul  3 21:42:57 openvpn[19830]: user 'bob' authenticated
<29>Jul  3 21:42:57 openvpn[66026]: bob/10.170.120.149:37849 MULTI_sva: pool returned IPv4=10.170.170.2, IPv6=(Not enabled)
<27>1 2021-07-03T22:17:01.074560-05:00 pfSense.example.com openvpn 66026 - - TLS Error: cannot locate HMAC in incoming packet from [AF_INET]175.16.199.1:34745
<36>1 2021-07-03T22:40:38.477134-05:00 pfSense.example.com openvpn 68813 - - user 'bob' could not authenticate.